alonzo/formal-spec/properties.tex

```diff
@@ -370,15 +369,16 @@ \section{Formal Properties}
 
     The $\fun{collectTwoPhaseScriptInputs}$
     function (see \ref{fig:functions:script2})
-    makes a list where each entry contains a Plutus script
+    constructs a list where each entry contains a Plutus script
     and the argumets passed to the interpreter for its evaluation.
 
     To show that $\fun{collectTwoPhaseScriptInputs}$ returns a list containing
     the same elements for both $tx$ and $tx'$,
     we observe that
     the set of elements from which this list is produced via $\fun{toList}$
     is generated using the following functions, as well as set comprehension
-    and list operations. We observe that these functions are deterministic
+    and list operations (which are known to be deterministic).
```

Comment:
I don't think we should be talking about functions being deterministic. We're doing a mathematical model, so there's no such thing as a non-deterministic function.

Reply:
yeah true